File: Enseignant.java
import java.util.ArrayList;
public class Enseignant extends Utilisateur {
	public ArrayList<Matiere> matieres;

	public Enseignant(int idUtilisateur, String nom, String prenom, String passwordHash, String emailPro, ArrayList<Matiere> matieres) {
		super(idUtilisateur, nom, prenom, passwordHash, emailPro);
		this.matieres = matieres;
	}

	public String toString(){
		return this.nom +" " + this.prenom + " (" + this.idUtilisateur + ")";
	}
}

File: Web/model/Enseignant.php
<?php
public class Enseignant {
	private $idUtilisateur;
	public $nom;
	public $prenom;
	private $passwordHash;
	public $emailPro;
	public $actif;
	public $idMatieres;

	public Enseignant(idUtilisateur, nom, prenom, passwordHash, emailPro, matieres) {
		$this->idUtilisateur = idUtilisateur;
		$this->nom = nom;
		$this->prenom = prenom;
		$this->passwordHash = passwordHash;
		$this->emailPro = emailPro;
		$this->actif = false;
		$this->matieres = matieres;
	}

}
?>
